db/schemas/employees.js

```diff
@@ -0,0 +1,60 @@
+db.createCollection('employees', {
+  validator: {
+    $jsonSchema: {
+      bsonType: 'object',
+      required: [
+        'firstName',
+        'lastName',
+        'address1',
+        'city',
+        'country',
+        'wage',
+        'position',
+        'isActive'
+      ],
+      properties: {
+        'firstName': {
+          bsonType: 'string',
+        },
+        'lastName': {
+          bsonType: 'string'
+        },
+        'address1': {
+          bsonType: 'string'
+        },
+        'address2': {
+          bsonType: 'string'
+        },
+        'city': {
+          bsonType: 'string'
+        },
+        'state': {
+          bsonType: 'string'
+        },
+        'country': {
+          bsonType: 'string'
+        },
+        'zipcode': {
+          bsonType: 'string'
+        },
+        'wage': {
+          bsonType: 'decimal'
+        },
+        'startDate': {
+          bsonType: 'date'
+        },
+        'username': {
+          bsonType: 'string'
+        },
```

Comment:
We will be using email as the username, so we won't be needing this username property. I already discussed with Aharon who did the employee front-end.

Reply:
Its in there per my last conversation with Theo and Tina. Waiting on confirmation from them that I can get rid of it.